app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 in_class: int,
+                 in_mark: int,
+                 in_brand: str) -> None:
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) specifically gives an example for such case, so perhaps the mainstream is going to adapt that style. 
Personally I advise to use separating each parameter into its own line, so that any future addition/deletion of parameter would give a clean ```git diff```
```
def __init__(
    self,
    in_class: int,
    in_mark: int,
    in_brand: str
) -> None:
```

Reply:
if you mentioned PEP - please give a certain PEP index. For example: "pls check  PEP 123"

Reply:
There is link in my comment above, if you press PEP8